lib/broadside/configuration/configuration.rb

```diff
@@ -41,6 +40,10 @@ def initialize
       @timeout = 600
     end
 
+    def ecs
+      @aws
```

Comment:
eh, not a huge fan of this. i'd prefer we have a separate object if we are going this route